pkg/storage/parse.go

```diff
@@ -29,6 +29,7 @@ func ParseBackend(rawURL string, options *BackendOptions) (*backup.StorageBacken
 		return nil, errors.Annotate(berrors.ErrStorageInvalidConfig, "empty store is not allowed")
 	}
 
+	rawURL = strings.ReplaceAll(rawURL, "+", "%2B")
```

Comment:
could we also add a unit test to `parse_test.go`

Reply:
Please add some comments.